src/packages/health/health.ts

```diff
@@ -29,4 +30,30 @@ export default (app: Express) => {
     const uptimeMinutes = process.uptime() / 60;
     res.json({ timestamp, uptimeMinutes });
   });
+
+  /**
+   * @swagger
+   * /version:
+   *   get:
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Version metrics
+   *         content:
+   *           application/json:
+   *             schema:
+   *                properties:
+   *                  gateway_version:
+   *                    description: The current version of the Aerie Gateway.
+   *                    type: string
+   *     summary: Get the current version of the Gateway and Database Schema
+   *     tags:
+   *       - Version
+   */
+  app.get('/version', async (_, res) => {
+    res.json({
+      gateway_version: getEnv().VERSION,
```

Comment:
@Mythicaeda I think it's better if we just call this key `version` rather than `gateway_version` - While I understand that it only reflects the version of the gateway service, to API *consumers* this represents/is a proxy for the version of the entire Aerie backend. If we someday come up with a better source for the "entire backend" version, we could replace the value & keep the key. With ^this key it forces consumers to understand that `gateway_version` is the proxy for backend version, which feels like an implementation detail.